### DISCUSSION TITLE
Unexpected Redundancy of an erasure set

### DISCUSSION THREAD

**Discussion by roeelavi** [minio/minio#13139] [Q&A] [answered]
## Expected Behavior
An erasure set of 4 drives should have a redundancy of 200% (2GB of total storage for 1GB of data) 
## Current Behavior
 Redundancy of 400% (4GB of total storage for 1GB of data)

## Steps to Reproduce (for bugs)
1. mounting the for drives sdc on /mnt/data1 sdd on /mnt/data2 sde on /mnt/data3 sdf on /mnt/data4
2. docker run --name minio -v /mnt/data1:/data1 -v /mnt/data2:/data2 -v /mnt/data3:/data3 -v /mnt/data4:/data4 -p 9000:9000 minio/minio server /data{1...4} 
3. create a bucket and add a 1GB file on it
4. the out put i get from df -h is that each drive contains 1GB of storage
Im not sure if its a bug or im just misunderstanding the concepts of erasure coding.
would love it if anyone has any explanation, and a solution (BTW I am using a virtual environment so I'm flexible with the number of drives)

**Comment by harshavardhana** [comment]
@roeelavi `du -sh /mnt/data*` and report it here

**Comment by klauspost** [comment, marked as answer]
I cannot reproduce. Uploading to 4 disk setup takes ~2x the space of the original, as expected.
